soh/soh/Enhancements/savestates.cpp

```diff
@@ -869,10 +869,12 @@ SaveStateReturn SaveStateMgr::AddRequest(const SaveStateRequest request) {
     switch (request.type) { 
         case RequestType::SAVE:
             requests.push(request);
+            return SaveStateReturn::SUCCESS;
             break;
```

Comment:
looks like we can remove all the `break`s in this switch/case because every path has a return

Reply:
Done.